tests/release/release.go

```diff
@@ -70,6 +53,28 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1108]test-release-service-happy-p
 			sa, err := framework.CommonController.GetServiceAccount(serviceAccount, managedNamespace)
 			return sa != nil && err == nil
 		}, pipelineServiceAccountCreationTimeout, defaultInterval).Should(BeTrue(), "timed out when waiting for the \"pipeline\" SA to be created")
+
+		// get the ec configmap to configure the policy and data sources
+		cm, err := framework.CommonController.GetConfigMap("ec-defaults", "enterprise-contract-service")
+		Expect(err).ToNot(HaveOccurred())
+		// the default policy source
+		ecPolicy = ecp.EnterpriseContractPolicySpec{
+			Description: "Red Hat's enterprise requirements",
+			Sources: []ecp.Source{
+				{
+					Name: "ec-policies",
+					Policy: []string{
+						cm.Data["ec_policy_source"],
+					},
```

Comment:
Not sure if go fmt will agree, but maybe it's nicer on a single line:

    Policy: []string{cm.Data["ec_policy_source"]}